### ISSUE TITLE
Generator is writing incorrect paths for queries and mutations

### ISSUE THREAD

**Issue by wKovacs64** [blitz-js/blitz#402]
### What is the problem?

`blitz generate` is creating queries and mutations one level too deep (under an additional model directory).

Example:

Received: `app\projects\queries\projects\getProjects.ts` 
Expected: `app\projects\queries\getProjects.ts`

### Steps to Reproduce: 

1. `blitz new myapp`
1. `cd myapp`
1. `blitz generate all project`

### Versions:

```
debug: monorepo
debug: pkgPath: C:\dev\contrib\blitz\node_modules\@blitzjs\cli

Windows 10 | win32-x64 | Node: v12.16.1

blitz: 0.8.1 (global)
blitz: 0.8.1 (local)
```

### Supporting Documentation

```
CREATE    app\projects\pages\projects\index.tsx
CREATE    app\projects\pages\projects\new.tsx
CREATE    app\projects\pages\projects\[id]\edit.tsx
CREATE    app\projects\pages\projects\[id].tsx
CREATE    app\projects\queries\projects\getProjects.ts
CREATE    app\projects\queries\projects\getProject.ts
CREATE    app\projects\mutations\projects\createProject.ts
CREATE    app\projects\mutations\projects\deleteProject.ts
CREATE    app\projects\mutations\projects\updateProject.ts
```

**Comment by aem** [comment]
Yeah this was known, good thought that it's a weird first experience though. 

My general thoughts here are that we currently try to make a lot of assumptions in `cli/generate` about file paths, but in reality that's hard because a couple generators (`app`, `pages`) have special rules that don't apply to the rest of the generators. I think I'd like to push that logic out into the generators and let the generator declare its own path via a set of heuristics rather than the one-size-fits-all approach that we currently have.

As a first pass I'll probably just hard-code some logic in the generator that special-cases `PageGenerator` and revisit the cleanup next week

**Comment by wKovacs64** [comment]
Yeah, I told a friend to check out Blitz and his immediate response was, "Sounds cool, but the tutorial doesn't even work." Oops. 😬

Generators declaring their own paths down the road with a temporary, hard-coded special case sounds reasonable. 👍 

Thanks!

**Comment by johncantrell97** [comment]
👍 Just ran into this as well when going through the tutorial to try creating a sample app to play with

**Comment by aem** [comment]
@johncantrell97 thanks for the report. I've been away from the computer for much of the weekend, but I'll take a look at this this afternoon. Sorry for the inconvenience here 😞

**Comment by flybayer** [comment]
@aem this used to work — do we know what caused it to break?

**Comment by aem** [comment]
@flybayer i’m not totally sure, i’ve got something mostly working locally though, should be able to put up a PR in a few minutes

**Comment by ganeshmani** [comment]
i  think the issue is with generator templates. 

**Previous Version**:
```import get__ModelNames__ from 'app/__modelNames__/queries/get__ModelNames__'```

**Modified Version**:
```import get__ModelNames__ from 'app/__modelNames__/queries/__modelNames__/get__ModelNames__'```

**Comment by flybayer** [comment]
@ganeshmani the import paths are correct, it's the path of the generated files that is wrong.

Broken: `app\projects\queries\projects\getProjects.ts`
Correct: `app\projects\queries\getProjects.ts`

**Comment by flybayer** [comment]
Fixed in blitz `0.8.2`!
